src/modules/revolutionPi/lib/README.md

```diff
@@ -0,0 +1,8 @@
+#Revolution Pi
+
+All files within this folder have been taken from external repositories hosted by KUNBUS GmbH.
+
+- [piControlIf.h](https://gitlab.com/revolutionpi/revpi-pitest/-/blob/master/include/piControlIf.h?ref_type=heads)
+- [piControlIf.c](https://gitlab.com/revolutionpi/revpi-pitest/-/blob/master/src/piControlIf.c?ref_type=heads)
+- [piControl.h](https://gitlab.com/revolutionpi/piControl/-/blob/master/piControl.h?ref_type=heads)
```

Comment:
these are not permalinks, meaning that when the master branch is updated, the files here will be out of date. The best will be to click on the permalink button on the top right in gitlab for each file to get the link to the specific version which won't change

Reply:
good point thanx